scripts/independent-pkg-version.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
```

Comment:
Nit: Can we rename this file as `bump-independent-pkg-version.js`?

Reply:
Alternatively, we can put these three scripts for version bumping into a separate folder